packages/arb-token-bridge-ui/src/util/cctp/fetchCCTP.ts

```diff
@@ -0,0 +1,46 @@
+import { ChainId } from '../networks'
+import { getAPIBaseUrl, sanitizeQueryParams } from '..'
+import { Response } from '../../pages/api/cctp/[type]'
+
+export type FetchParams = {
+  walletAddress: string
+  l1ChainId: ChainId
+}
+
+async function fetchCCTP({
+  walletAddress,
+  l1ChainId,
```

Comment:
Let's change to more closely match the CCTP terminology:

```suggestion
  sourceChainId,
```

Reply:
Updated